pytransform3d/transform_manager/__init__.py

```diff
@@ -5,8 +5,11 @@
 from ._transform_graph_base import TransformGraphBase
 from ._transform_manager import TransformManager
 from ._temporal_transform_manager import (StaticTransform,
-                                          TemporalTransformManager)
+                                          TimeVaryingTransform,
+                                          TemporalTransformManager,
+                                          NumpyTimeseriesTransform)
 
 
 __all__ = ["TransformGraphBase", "TransformManager",
-           "TemporalTransformManager", "StaticTransform"]
+           "TemporalTransformManager", "TimeVaryingTransform",
+           "StaticTransform", "NumpyTimeseriesTransform"]
```

Comment:
All of these classes should also be available in the API documentation. Please add them [here](https://github.com/dfki-ric/pytransform3d/blob/master/doc/source/api.rst)

Reply:
Done